platform-hub-api/app/services/encryptor_service.rb

```diff
@@ -0,0 +1,15 @@
+class EncryptorService
+  def initialize(secret_key_base)
+    @encryptor = ActiveSupport::MessageEncryptor.new(secret_key_base)
+  end
+
+  def encrypt(plaintext)
+    @encryptor.encrypt_and_sign(plaintext)
+  end
+
+  def decrypt(encrypted_data)
+    @encryptor.decrypt_and_verify(encrypted_data)
+  rescue ActiveSupport::MessageVerifier::InvalidSignature
+    nil
```

Comment:
True, we probably don't, in the normal case. But in the abnormal case, e.g. a bug in the consumer of this code, we'd end up with potential null pointer exceptions (or handling `nil` values when there should be). But I guess at that point we can debug closer.